src/SFA.DAS.ApplyService.InternalApi.UnitTests/RoatpGatewayCriminalComplianceChecksControllerTests.cs

```diff
@@ -0,0 +1,204 @@
+                                new Option
+                                {
+                                    Value = "Yes",
+                                    FurtherQuestions = new List<Question>
+                                    {
+                                        new Question
+                                        {
+                                            QuestionId = "CC-22.1",
+                                            Input = new Input
+                                            {
+                                                Type = "Text"
+                                            }
+                                        }
+                                    }
+                                },
+                                new Option
+                                {
+                                    Value = "No"
+                                }
+                            }
+                        }
+                    }
+                },
+                PageOfAnswers = new List<PageOfAnswers>
+                {
+                    new PageOfAnswers
+                    {
+                        Answers = new List<Answer>
+                        {
+                            new Answer
+                            {
+                                QuestionId = "CC-22",
+                                Value =  "No"
+                            }
+                        }
+                    }
+                }
+            };
+
+            _qnaApiClient.Setup(x => x.GetPageBySectionNo(It.IsAny<Guid>(), RoatpWorkflowSequenceIds.CriminalComplianceChecks,
+                                                          RoatpWorkflowSectionIds.CriminalComplianceChecks.ChecksOnYourOrganisation, pageDetails.PageId))
+                                .ReturnsAsync(qnaPageWithQuestion);
+
+            var result = _controller.GetCriminalComplianceQuestionDetails(Guid.NewGuid(), gatewayPageId).GetAwaiter().GetResult();
+
+            var objectResult = result as OkObjectResult;
+            objectResult.Should().NotBeNull();
+            var criminalComplianceDetails = objectResult.Value as CriminalComplianceCheckDetails;
+            criminalComplianceDetails.Should().NotBeNull();
+            criminalComplianceDetails.PageId.Should().Be(qnaPageWithQuestion.PageId);
+            criminalComplianceDetails.QuestionText.Should().Be(qnaPageWithQuestion.Questions[0].Label);
+            criminalComplianceDetails.QuestionId.Should().Be(qnaPageWithQuestion.Questions[0].QuestionId);
+            criminalComplianceDetails.Answer.Should().Be(qnaPageWithQuestion.PageOfAnswers[0].Answers[0].Value);
+            criminalComplianceDetails.FurtherQuestionId.Should().BeNull();
+            criminalComplianceDetails.FurtherAnswer.Should().BeNull();
+        }
+
+        [Test]
+        public void Get_question_details_retrieves_question_details_and_answer_with_further_question()
```

Comment:
Similar comment around duplication between tests.

Reply:
Done

Reply:
It's not showing any changes in this file?

Reply:
Oops, thought the comment was related to the same tests as above

Reply:
That's updated now